src/forward_path.ml

```diff
@@ -14,6 +14,17 @@ let equal a b = match a, b with
   | Forward_path a, Forward_path b -> Path.equal a b
   | _, _ -> false
 
+let compare a b =
+  let inf = (-1) and sup = 1 in
+  match a, b with
+  | Postmaster, Postmaster -> 0
+  | Domain a, Domain b -> Domain.compare a b
+  | Forward_path a, Forward_path b -> Path.compare a b
+  | Postmaster, _ -> sup
```

Comment:
`compare a b` is `a - b` so `sup` should be `-1` and `inf` 1 ?

Reply:
Depends on which order we want. I'm not a big fan of such ppx and I would not like to add a new dependency on `colombe`. At least, we should provide tests on these functions to check expected order.

Reply:
PPX are tested and don't break when the code changes. `ppx_deriving` is common and the user might use it already.